docs/source/release_notes.rst

```diff
@@ -17,6 +20,7 @@ Future Release
         * Add ``__setitem__`` method to overload ``add_dataframe`` method on EntitySet (:pr:`1862`)
         * Temporarily restrict woodwork max version (:pr:`1872`)
         * Split Datetime and LatLong primitives into separate files (:pr:`1861`)
+        * Update to add support for pandas version 1.4.0 (:pr:`1881`)
```

Comment:
Let's put this in the enhancement category

Reply:
done!